ods_ci/tests/Tests/0100__platform/0101__deploy/0102__auth_providers_and_rbac/0103__auth_crd.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Documentation       Test Suite for Auth CRD functionality verification
+
+Resource            ../../../../Resources/ODS.robot
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource            ../../../../Resources/RHOSi.resource
+
+Suite Setup         Auth Crd Suite Setup
+Suite Teardown      Auth Crd Suite Teardown
+
+
+*** Test Cases ***
+Verify Auth Crd
+    [Documentation]    Happy path to cover the functionality of the Auth CRD
+    [Tags]      Smoke
+    ...         Operator
+    ...         RHOAIENG-18846
+    ...         ExcludeOnODH
+    ${rc}       ${odhdashboardconfig_groups}=    Run And Return Rc And Output
+    ...    oc get odhdashboardconfig odh-dashboard-config -n redhat-ods-applications -o jsonpath='{.spec.groupsConfig}'
+    Should Be Equal As Integers     ${rc}       0
+    ${rc}       ${auth_cr_groups}=      Run And Return Rc And Output
+    ...                     oc get auth auth -o jsonpath='{.spec}'
+    Should Be Equal As Integers     ${rc}       0
+    ${auth_cr_groups_str}=      Remove String       ${auth_cr_groups}   [   ]
+    IF    """${odhdashboardconfig_groups}""" != """${auth_cr_groups_str}"""
+        Fail        User Groups on Auth Cr and OdhDashboardConfig DO NOT match
+    ELSE
+        Log To Console      User Groups on Auth Cr and OdhDashboardConfig match
+    END
+    Get User Groups From Auth Cr And Check Rolebinding Exists       adminGroups     admingroup-rolebinding
```

Comment:
For admingGroups also ClusterRoleBinding `admingroupcluster-rolebinding` is created. For allowedGroups only rolebinding is being created.

Reply:
Thanks Marian, I will add that check too